client/reducers/reducer.js

```diff
@@ -11,7 +12,7 @@ const INITIAL_STATE = {
     positionY: 4
   },
   board: levels[1],
-  commandQueue: [], // commands are the same as the action types. e.g. 'MOVE_FORWARD'
+  commandQueue: [], // commands are the same as the action types. e.g. a.MOVE_FORWARD
```

Comment:
Looks like you've done a change all on ' with a.    . Some comments have been messed up